client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -527,6 +529,40 @@ export class PluginsList extends Component {
 		}
 	};
 
+	updateAllPluginsDialog = () => {
+		const { pluginUpdateCount, translate } = this.props;
+
+		const siteDetails = this.props.selectedSiteSlug ? this.props.selectedSiteSlug : 'all sites';
+
+		const translationArgs = {
+			args: { pluginUpdateCount },
+			count: pluginUpdateCount,
+		};
+
+		acceptDialog(
+			<div>
+				<span>
+					{ translate(
+						'You are about to update %(pluginUpdateCount)d plugin on %(siteDetails)s',
```

Comment:
Strings like these are hard to translate - check the comment here for more details - https://github.com/Automattic/wp-calypso/pull/67799#pullrequestreview-1107285580.

We can keep these translations separate since this is newly added.

Reply:
Thanks for the heads up and reference on this. I've made some updates based on what was implemented in the linked comment